README.md

```diff
@@ -1,17 +1,169 @@
+|-------------|----------------------------------------------------|-------------|---------|----------|
+| common_tags | Map of common tags for all taggable AWS resources. | map(string) | {}      | No       |
+| aws_region  | AWS region to deploy resources into.               | string      |         | Yes      |
+
+### Backup configurations
+
+| Name                   | Description                         | Type   | Default     | Required |
+|------------------------|-------------------------------------|--------|-------------|----------|
+| deploy_backup          | Deploy backup module.               | bool   | true        | No       |
+| backup_schedule        | Cron expression for the backup job. | string | "0 0 * * *" | No       |
+| backup_retention_count | Number of backups to keep.          | number | 7           | No       |
+
+### Load balancer & TLS
+
+| Name                      | Description                                                                                                                        | Type         | Default                               | Required |
+|---------------------------|------------------------------------------------------------------------------------------------------------------------------------|--------------|---------------------------------------|----------|
+| lb_internal               | Whether the load balancer will be internal or public.                                                                              | bool         | false                                 | No       |
+| lb_deregistration_delay   | Time, in seconds, for GraphDB LB target group to wait before changing the state of a deregistering target from draining to unused. | string       | 300                                   | No       |
+| lb_health_check_path      | The endpoint to check for GraphDB's health status.                                                                                 | string       | "/rest/cluster/node/status"           | No       |
+| lb_health_check_interval  | Interval in seconds for checking the target group health check.                                                                    | number       | 10                                    | No       |
+| lb_tls_certificate_arn    | ARN of the TLS certificate used for the TLS listener on the load balancer.                                                         | string       | null                                  | No       |
+| lb_tls_policy             | TLS security policy on the listener.                                                                                               | string       | "ELBSecurityPolicy-TLS13-1-2-2021-06" | No       |
+| allowed_inbound_cidrs_lb  | List of CIDR blocks to permit inbound traffic to the load balancer.                                                                | list(string) | null                                  | No       |
```

Comment:
shouldn't `allowed_inbound_cidrs_lb` be required ? otherwise you won't be able to open the WB.